support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.jsx

```diff
@@ -144,19 +154,15 @@ function updatePayerState(token: Object, setState: (UsState | CaState | null) =>
   if (state) {
     setState(state);
     return true;
-  } else {
-    logException('Missing address_state in payment request token');
-    return false;
   }
+  logException('Missing address_state in payment request token');
+  return false;
+
 }
 
-// Calling the complete function will close the pop up payment window
-const onComplete = (complete: Function) => (res: PaymentResult) => {
+const onComplete = () => (res: PaymentResult) => {
   if (res.paymentStatus === 'success') {
     trackComponentClick('apple-pay-payment-complete');
-    complete('success');
-  } else if (res.paymentStatus === 'failure') {
```

Comment:
no longer any need to check for failure here, as we're not using the browser popup